Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -63,8 +63,12 @@ function Initialize-AzureSubscription {
     if ($Endpoint.Auth.Scheme -eq 'ServicePrincipal' -and !$script:azureModule -and (Get-Command -Name "Clear-AzureRmContext" -ErrorAction "SilentlyContinue")) {
         Write-Host "##[command]Clear-AzureRmContext -Scope Process"
         $null = Clear-AzureRmContext -Scope Process
-        Write-Host "##[command]Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue"
-        $null = Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue
+    }
+
+    if (Get-Command -Name "Disable-AzureRmContextAutosave" -ErrorAction "SilentlyContinue") 
+    {
+        Write-Host "##[command]Disable-AzureRmContextAutosave -ErrorAction SilentlyContinue"
```

Comment:
set error action to stop and put this under a try-catch

Reply:
done.